rc/tools/git.kak

```diff
@@ -69,12 +69,13 @@ define-command -params 1.. \
             grep
     } -shell-script-candidates %{
     if [ $kak_token_to_complete -eq 0 ]; then
-        printf "add\nrm\nreset\nblame\ncommit\ncheckout\ndiff\nhide-blame\nhide-diff\nlog\nnext-hunk\nprev-hunk\nshow\nshow-branch\nshow-diff\ninit\nstatus\nupdate-diff\ngrep\n"
+        printf "add\nrm\nreset\nblame\ncommit\ncheckout\ndiff\nhide-blame\nhide-diff\nlog\nnext-hunk\nprev-hunk\nshow\nshow-branch\nshow-diff\ninit\nstatus\nupdate-diff\ngrep\nedit\n"
     else
         case "$1" in
             commit) printf -- "--amend\n--no-edit\n--all\n--reset-author\n--fixup\n--squash\n"; git ls-files -m ;;
             add) git ls-files -dmo --exclude-standard ;;
-            rm|grep) git ls-files -c ;;
+            rm) git ls-files -c ;;
+            grep|edit) git ls-files -c --recurse-submodules ;;
```

Comment:
In my config I also have an alternative command that completes from top-level instead of cwd (`git ls-files "$(git rev-parse --show-toplevel)"`) that is useful occasionally. Let's worry about that later, not yet sure about the interface (separate command / flag / GIT_WORK_TREE option)

Reply:
yeah using `$PWD` is the correct default, not `git rev-parse --show-toplevel`.
Same for other git commands (this allows relative pathspecs for example).